### ISSUE TITLE
Latest version lags extreme after a clouple of minutes

### ISSUE THREAD

**Issue by phil2sat** [paulov-t/SIT.Core#63]
**Describe the bug**
Game extremely Laggy after a couple of minutes

**To Reproduce**
Start Aki.Server on other PC (Ryzen4700u 16gb RAM Nvme)
Connect via AKI.Launcher
Start playing as Single Player on Fabrik
First on ro two kills went fine after that, jumping, opening/closing doors, killing Bots will take up to 10 seconds

**Expected behavior**
Instant action

**Screenshots**

***Game Version***
0.13.0.4.23122

**SIT.Core Version:**
1.2.8541.17221

**SPT-Aki Version:**
3.5.7-PRE

**Additional context**
Want to play with my Family, so i'm testing

**Comment by pi0t3r2** [comment]
I am going to try this out, so I'll let you know later.

**Comment by phil2sat** [comment]
Im playing over LAN with a Ping of less than one ms. So i exclude that
Even my Internet is a 500mbit/50mbit Fiber line if i would use it, maybe if the internals runs smooth
But if i only knowing how i would change the spawn amount ill could give it a try

**Comment by pi0t3r2** [comment]
> 
Yeah the thing it says on the top letft isn't actually true and on Radmin when I played with my friend and there was like around 20 bots then it  was lagging a lot so I decided to check ping with my friend on Radmin and there was like around 500ms.

**Comment by pi0t3r2** [comment]
Ok so I tweaked the settings on KMC server value modifier and no issues with running the server i decreased the max bot amount but I'' have to see if the settings are actually working.

**Comment by pi0t3r2** [comment]
Omg no way it's actually working, I think it's working but boss chance spawns doesn't work both reshale and the goons spawned. Check magazine animation is like delayed by 1/4 second

**Comment by pi0t3r2** [comment]
which is pretty dang good

**Comment by phil2sat** [comment]
I'll testing right now, edited the 
Aki_Data\Server\database\locations\factory4_day\base.json
searched for waves and set them all to 1 like
```  "waves": [
    {
      "BotPreset": "normal",
      "BotSide": "Savage",
      "SpawnPoints": "BotZone",
      "WildSpawnType": "assault",
      "isPlayers": false,
      "number": 1,
      "slots_max": 1,
      "slots_min": 1,
      "time_max": -1,
      "time_min": -1
    },
    {
      "BotPreset": "normal",
      "BotSide": "Savage",
      "SpawnPoints": "BotZone",
      "WildSpawnType": "assault",
      "isPlayers": false,
      "number": 1,
      "slots_max": 1,
      "slots_min": 1,
      "time_max": 500,
      "time_min": 500
    },
    {
      "BotPreset": "hard",
      "BotSide": "Savage",
      "SpawnPoints": "BotZone",
      "WildSpawnType": "assault",
      "isPlayers": true,
      "number": 1,
      "slots_max": 1,
      "slots_min": 1,
      "time_max": 1000,
      "time_min": 1000
    },
    {
      "BotPreset": "normal",
      "BotSide": "Savage",
      "SpawnPoints": "BotZone",
      "WildSpawnType": "assault",
      "isPlayers": true,
      "number": 1,
      "slots_max": 1,
      "slots_min": 0,
      "time_max": 1500,
      "time_min": 1500
    },
    {
      "BotPreset": "normal",
      "BotSide": "Savage",
      "SpawnPoints": "BotZone",
      "WildSpawnType": "assault",
      "isPlayers": true,
      "number": 1,
      "slots_max": 1,
      "slots_min": 1,
      "time_max": 2000,
      "time_min": 2000
    },
    {
      "BotPreset": "hard",
      "BotSide": "Savage",
      "SpawnPoints": "BotZone",
      "WildSpawnType": "assault",
      "isPlayers": true,
      "number": 1,
      "slots_max": 1,
      "slots_min": 1,
      "time_max": 2500,
      "time_min": 2500
    }

**Comment by phil2sat** [comment]
Yea, doesnt work we all are Invincible after that

**Comment by pi0t3r2** [comment]
But if you're realy desperate host can try disabling boss spawns and through kmc you can tweak the loot quality for pmcs and stuff

**Comment by pi0t3r2** [comment]
Also there is this one thing that I hate so it's about the spawns you can't spawn together and it's insanely hard for my friend that is new to the game and he doesn't know any of the maps to get to me without dying. Do you know how to change spawns on maps so there is a only one spawn for example

**Comment by pi0t3r2** [comment]
I'll try other maps than customs now like lighthouse.

**Comment by pi0t3r2** [comment]
Hmm it's pretty weird, it says that boss spawn chances for example for the reshala on customs is 28 but it's a guarantee spawn

**Comment by Jailbreak0726** [comment]
Same problem here but what weird for me is I'm in fact testing 2 devices on the same console with virtual lan, so there's technically no delays between host and client, and it occurs randomly, sometimes the experience is great while some time doesn't, also seems to me it doesn't relate to ram issue for delays may occur in the first minute of the first round

**Comment by phil2sat** [comment]
Yea, i cant figure out whats going on
from
 - bots without body, only the gear visible
 - bots not moving after joining
 - bots and i invincible
 - jumping, doin actions are delayed, killing me, bots also
 really weird gets instantly if i heal me after i get wounded
its like bufferbloat, the beginning is realtime, the more actions happen, the more gets buffered/delayed. its hiw it feels or my dumb explanation.

also the server console spams a huge amount of id's while playing. is it possible to force this to off. btw log requests is false

**Comment by SlejmUr** [comment]
The Ids are the aid/userid/id who joined to the server.
You can delete in the console.log

**Comment by phil2sat** [comment]
But how, for me it looks like loot or inventory IDs and even if i set log requests to false in http.json theyre like waterfalls on server console if im playing, maybe simply the console spam sliws the server down, just an idea.

**Comment by phil2sat** [comment]
Compiled actual SIT.Core.dll from source which seems to work better in the first test, healed a lot and no lag.
Further testing.....

**Comment by pi0t3r2** [comment]
So for now what works absolutely amazing is factory, labs and interchange with bosses set to on and the rest is running good without boss spawns I've managed to headshot a scav from a 200 meters with VPO-215 so it seeems to work good. Deaths are a bit delayed but it's good

**Comment by pi0t3r2** [comment]
If Paulov would include a system that deletes bots when you are not in a specific spawn zone or close to and also add some configuration it would be great  and also disable secondary bosses . Then the performance would be better but there are also some other things that are overloading the server.

**Comment by phil2sat** [comment]
> nasa internet and the ability to portforward
All this is fine ports are forwarded, but actually im playing on LAN. Since i want to exclude as much as possible.
Some matches work fine, some not, some even at the beginning not, yea its random.

I dont know if ist possible, but as i see my Server is really idling while playing at about 5% and around 6.2G of Ram
- Server generated Bots.
But i think its not possible

**Comment by Jailbreak0726** [comment]
I think about the spawning limitations there are mods that allows only spawning new npcs that withia certain distance with the host/player, however I forgot what exactly the name for it and whether its will conflict with coopmod when installed

**Comment by pi0t3r2** [comment]
So I tested customs with bots set to low with bosses on and I must say that It works pretty good I managed to kill reshala, but then when I was going to smugler's boat extract I was staying close to the fence that's like close to dorms and I stepped on a barbed wire that one shot killed me

**Comment by phil2sat** [comment]
Switched Server to Linux, now running under "wine"
So i played a while with developer Bandana and it seems if i get less damage its less lagging. Last 10 Rounds all fine
Next try let my Kids join...
